Dockerfile

```
@@ -0,0 +1,28 @@
+# Set environment variables
+ENV PYTHONUNBUFFERED=1 \
+    PYTHONDONTWRITEBYTECODE=1
+
+# Copy MCP server code
+COPY fast-markdown-mcp/ /app/
+
+# Install dependencies directly without using a virtual environment
+RUN pip install --no-cache-dir -e .
+
+# Create necessary directories
+RUN mkdir -p /app/storage/markdown
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)